pkg/schedulingtypes/resources.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
How about move this to `schedulingtypes/utils`?

Reply:
This is not a utility file, but inherent component of the main functionality.